terraform/modules/standard_application_fargate/ecs_service.tf

```
@@ -0,0 +1,78 @@
+data "aws_iam_policy_document" "service_assume_role" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type = "Service"
+      identifiers = [
+        "ecs.amazonaws.com",
+      ]
+    }
+
+    actions = ["sts:AssumeRole"]
+  }
+}
+
+data "aws_iam_policy_document" "service_policy" {
+  statement {
+    effect = "Allow"
+
+    resources = ["*"]
+
+    actions = ["logs:*"]
+  }
+
+  statement {
+    effect = "Allow"
+
+    resources = ["*"]
+
+    actions = [
+      "ecs:CreateCluster",
+      "ecs:DeregisterContainerInstance",
+      "ecs:DiscoverPollEndpoint",
+      "ecs:Poll",
+      "ecs:RegisterContainerInstance",
+      "ecs:StartTelemetrySession",
+      "ecs:UpdateContainerInstancesState",
+      "ecs:Submit*",
+      "ecr:GetAuthorizationToken",
+      "ecr:BatchCheckLayerAvailability",
+      "ecr:GetDownloadUrlForLayer",
+      "ecr:BatchGetImage",
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+      "ec2:Describe*",
+    ]
+  }
+}
+
+
+resource "aws_iam_role" "service_role" {
```

Following the same as other comment, we can cut `role` from name
```suggestion
resource "aws_iam_role" "service" {
```

---

:up: